src/keycloak/keycloak_admin.py

```diff
@@ -2810,3 +2812,52 @@ def create_client_authz_client_policy(self, payload, client_id):
             data=json.dumps(payload),
         )
         return raise_error_from_response(data_raw, KeycloakPostError, expected_codes=[201])
+
+    def get_composite_client_roles_of_group(self, client_id, group_id):
```

Comment:
All of the new methods are missing docstrings, they are required.

Reply:
Ok I have added the docstrings.